# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/DOMLocalization.h
# Commit: 92b87b0f776c
# Full Hash: 92b87b0f776c3049499038d380190351e2371324
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r 878b052ebeaa -r 92b87b0f776c dom/l10n/DOMLocalization.h
--- a/dom/l10n/DOMLocalization.h	Tue Aug 03 16:25:10 2021 +0000
+++ b/dom/l10n/DOMLocalization.h	Tue Aug 03 16:25:10 2021 +0000
@@ -14,6 +14,7 @@
 #include "mozilla/dom/L10nMutations.h"
 #include "mozilla/dom/L10nOverlaysBinding.h"
 #include "mozilla/dom/LocalizationBinding.h"
+#include "mozilla/dom/PromiseNativeHandler.h"
 
 // XXX Avoid including this here by moving function bodies to the cpp file
 #include "nsINode.h"